### PR TITLE
Add sk-SK.json (slovak) localization file

### PR DESCRIPTION
**Pull request by matus123**
# Issue

#2375 

Adding slovak localization file.

I tested it with manual build from source.

## Tasklist

 - [ ] Add tests
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Generally use squash merge to rebase and clean comments before merging
 - [ ] Update the [changelog](CHANGELOG.md)

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

**Comment by kevinkreiser** [comment]
@matus123 thank you so much for this!

File: locales/sk-SK.json [hunk 1]
```diff
@@ -0,0 +1,1895 @@
...
+        ],
+        "5": [
+          "Otočte se vlevo přes Devonshire Road a zůstaňte na Jonestown Road, U.S. 22."
+        ]
+      }
+    },
+    "verbal_multi_cue": {
+      "phrases": {
+        "0": "<CURRENT_VERBAL_CUE> Potom <NEXT_VERBAL_CUE>",
+        "1": "<CURRENT_VERBAL_CUE> Then, in <LENGTH>, <NEXT_VERBAL_CUE>"
```
Comment (danpaz) [inline locales/sk-SK.json:1845]: Should this be `<CURRENT_VERBAL_CUE> Potom do <LENGTH>, <NEXT_VERBAL_CUE>` ?

**Comment by danpaz** [comment]
Thank you @matus123! @dgearhart can you reconcile this with #2378 once it's merged?

**Comment by dgearhart** [comment]
> Thank you @matus123! @dgearhart can you reconcile this with #2378 once it's merged?

If this one (#2376) is merged first then yes i can merge into mine (#2378) and generate place holders for the new roundabout phrases. Maybe @matus123 could fill in the new phrase after I merge #2378

File: locales/sk-SK.json (hunk 1 above)
Comment (matus123) [reply locales/sk-SK.json:1845]: My bad, i will fix this

**Comment by matus123** [comment]
I can create new PR where i replace newly created placeholders from #2378 after it gets merged.

**Pull request merged**